### PR TITLE
Docs/aspect models updates 3.2

### PR DESCRIPTION
**Pull request by jacewski-bosch**
This PR updates all Aspect Models for CX Release 3.2
Links to new and updated standardization dDocuments are still a todo (not available yet)

### CODE REVIEW

**Review by mhellmeier — CHANGES_REQUESTED** [review]
The merge conflicts should be solved.

**Comment by jacewski-bosch** [comment]
@mhellmeier fixed merge conflicts and thus the other issue as well

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 1]
```diff
@@ -294,7 +300,79 @@ Github Link to semantic data model: [https://github.com/eclipse-tractusx/sldt-se
 }
 ```
 
-> Please note that if a just-in-sequence part is also a serialized part SerialPartTypization should be used instead.
+> Please note that if a just-in-sequence part is also a serialized part SerialPart should be used instead.
+
+#### Submodel TractionBatteryCode
```
Comment (mhellmeier) [inline docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:305]: If you look into this markdown file, `TractionBatteryCode` can be found two times. Could you please remove one of the duplicates?

**Review by mhellmeier — CHANGES_REQUESTED** [review]
Great, @jacewski-bosch ! Could you please update the Changelog and add one or two sentences to describe the changes (like renaming from `SerialPartTypization` to `SerialPart` etc.)?

Changelog file: https://github.com/catenax-ng/eclipse-tractusx.github.io/blob/docs/trace-release-3.2/docs/kits/Traceability%20Kit/page_changelog.md

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md (hunk 1 above)
Comment (jacewski-bosch) [reply docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:305]: removed

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 2]
```diff
@@ -504,18 +522,19 @@ The following conventions apply for the endpoint:
...
+> :raised_hand: **Backend Data Service for Submodels**
+According to CX-0002, the backend data service identified via `href`and the filter criteria in `subprotocolBody` MUST be conformant to the Asset Administration Shell Profile SSP-003 of the Submodel Service Specification and must at least support the logical operation GetSubmodel. In release 3.2, only the logical parameter Content=Value must be supported (via path suffixes like in "/submodel/$value"). This might change in later Catena-X releases.
 
 With this approach, the EDC asset structure must no longer follow the "one EDC asset per submodel" rule (as in Release 3.1 and before), but gives data providers more flexibility how to create EDC assets for their digital twins and submodels based on how they use `{path}`.
 
-**Option 1: Same EDC Asset Structure as in Release 3.1**
+###### Option 1: Same EDC Asset Structure as in Release 3.1
```
Comment (jacewski-bosch) [inline docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:537]: @mhellmeier please check if this header has the correct level

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 3]
```diff
@@ -561,9 +580,9 @@ Here's an example how such a submodel descriptor could look like:
 ]
 ```
 
-In this example, the `path` part in the `href` is empty, as the EDC asset referenced in `subprotocolBody` directly points to a service returning the correct submodel (set up correctly with its dataAddress in the data provider's EDC). 
+In this example, the `path` part in the `href` is empty, as the EDC asset referenced in `subprotocolBody` directly points to a service returning the correct submodel (set up correctly with its dataAddress in the data provider's EDC).
 
-**Option 2: EDC Asset Structure on Catalog Part Level**
+###### Option 2: EDC Asset Structure on Catalog Part Level
```
Comment (jacewski-bosch) [inline docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:585]: @mhellmeier please check if this header has the correct level

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 4]
```diff
@@ -605,11 +624,11 @@ Here's an example how such a submodel descriptor could look like:
 ]
 ```
 
-The path part of the `href` property contains the information for the backend data service which digital twin's submodel to return while the EDC asset id is used for several endpoints. The path part here is just an example as it depends on the type of backend data service the data provider uses. 
+The path part of the `href` property contains the information for the backend data service which digital twin's submodel to return while the EDC asset id is used for several endpoints. The path part here is just an example as it depends on the type of backend data service the data provider uses.
 
 The above options are only two examples how a submodel's endpoint can be created. As long as it's compliant with the above conventions (including CX-0002) a data provider can also use any other EDC asset structure.
 
-**Data Consumption with AAS Submodel Descriptor Endpoints**
+###### Data Consumption with AAS Submodel Descriptor Endpoints
```
Comment (jacewski-bosch) [inline docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:631]: @mhellmeier please check if this header has the correct level

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md (hunk 2 above)
Comment (mhellmeier) [reply docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:537]: Header level is correct!

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md (hunk 3 above)
Comment (mhellmeier) [reply docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:585]: Header level is correct!

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md (hunk 4 above)
Comment (mhellmeier) [reply docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:631]: Header level is correct!

**Review by mhellmeier — APPROVED** [review]
LGTM

**Pull request merged**